snapshot-tests/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
I'm not sure why it wasn't required before. The permission was required for a long, long time: https://github.com/aws-amplify/aws-sdk-android/blob/ce33403116f27c1524796ddf96fd429d4316d433/aws-android-sdk-s3/src/main/AndroidManifest.xml#L4 

It was never merged automatically, because the `libs.aws.android.sdk.s3` dependency is added only to the test-apk, which AFAIR can't have its own manifest? Or resources at all 🤔 I don't remember what precisely didn't work. 

Anyway, I hope this is the fix for the CI failure: 
```kotlin
java.lang.SecurityException: ConnectivityService: Neither user 10063 nor current process has android.permission.ACCESS_NETWORK_STATE.
at android.os.Parcel.readException(Parcel.java:1599)
at android.os.Parcel.readException(Parcel.java:1552)
at android.net.IConnectivityManager$Stub$Proxy.getActiveNetworkInfo(IConnectivityManager.java:974)
at android.net.ConnectivityManager.getActiveNetworkInfo(ConnectivityManager.java:646)
at com.amazonaws.mobileconnectors.s3.transferutility.TransferNetworkConnectionType.isConnected(TransferNetworkConnectionType.java:82)
at com.amazonaws.mobileconnectors.s3.transferutility.TransferRecord.checkPreferredNetworkAvailability(TransferRecord.java:375)
at com.amazonaws.mobileconnectors.s3.transferutility.TransferRecord.start(TransferRecord.java:188)
at com.amazonaws.mobileconnectors.s3.transferutility.TransferUtility.submitTransferJob(TransferUtility.java:982)
at com.amazonaws.mobileconnectors.s3.transferutility.TransferUtility.upload(TransferUtility.java:586)
at com.amazonaws.mobileconnectors.s3.transferutility.TransferUtility.upload(TransferUtility.java:537)
at com.amazonaws.mobileconnectors.s3.transferutility.TransferUtility.upload(TransferUtility.java:491)
at com.airbnb.lottie.snapshots.utils.HappoSnapshotter$uploadDeferred$2.invokeSuspend(HappoSnapshotter.kt:136)
at com.airbnb.lottie.snapshots.utils.HappoSnapshotter$uploadDeferred$2.invoke(HappoSnapshotter.kt)
at com.airbnb.lottie.snapshots.utils.HappoSnapshotter$uploadDeferred$2.invoke(HappoSnapshotter.kt)
```

Reply:
Odd, I would expect the manifest to get merged 🤷